go.mod

```diff
@@ -13,7 +13,7 @@ require (
 	github.com/k8stopologyawareschedwg/deployer v0.21.0
 	github.com/k8stopologyawareschedwg/noderesourcetopology-api v0.1.2
 	github.com/k8stopologyawareschedwg/podfingerprint v0.2.2
-	github.com/k8stopologyawareschedwg/resource-topology-exporter v0.16.1
+	github.com/k8stopologyawareschedwg/resource-topology-exporter v0.19.0
```

Comment:
please let's consume v0.20.3. Changes between v0.19.0 and v0.20.3 should not create any major disruption: https://github.com/k8stopologyawareschedwg/resource-topology-exporter/compare/v0.19.0...v0.20.3